app/views/foods/_food.html.erb

```diff
@@ -0,0 +1,14 @@
+<tr class='food-row'>
+    <td><%= food.name %></td>
+    <td><%= food.measurement_unit %></td>
+    <td>$<%= food.price %></td>
+    <% if with_actions %>
+        <td class='action-buttons'>
+            <%= link_to "Modify", edit_food_path(food), class: 'info-bg' if can? :update, food %>
+            <%= link_to "Delete", food_path(food), data: {
+                    turbo_method: :delete,
+                    turbo_confirm: 'Are you sure you want to delete this food?'
+                }, class: 'danger-bg' if can? :destroy, food %>
+        </td>
+    <% end %>
```

Comment:
- [ ] When trying to delete a Food that have recipes I got **InvalidForeignKey** error (see image).
![Screenshot from 2023-12-23 18-07-49](https://github.com/patriciachrysy/Recipe-App/assets/91142432/3d13f114-b2f5-4bff-9023-6c8b90273c62)
This error can be fixed by adding `dependent: :destroy` to the the Food and recipe_foods in the recipe model
```rb
    has_many :recipe_foods, dependent: :destroy
```

Reply:
Fixed. Successfully deleted Food.
![image](https://github.com/patriciachrysy/Recipe-App/assets/93598189/a54a4580-abc9-4577-b4e2-deda6e8f254a)